index.js

```diff
@@ -216,6 +205,25 @@ const ContainerService = {
     }
   },
 
+  _createIdentityState() {
+    return {
+      hiddenTabUrls: [],
+      openTabs: 0
+    };
+  },
+
+  _maybeRemapTabs(userContextId) {
```

Comment:
Can we change to `_remapTabsIfMissing` so it's more obvious what the method is doing everywhere it's called?